.github/workflows/test-cosmwasm-onchain.yml

```
@@ -0,0 +1,233 @@
+            kill $MANTRACHAIND_PID || true
+          fi
+          # Also kill any remaining mantrachaind processes
+          pkill mantrachaind || true
+
+      # Upload test artifacts
+      - name: Upload test artifacts
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: test-results-${{ github.run_number }}
+          path: |
```

actionlint (1.7.7)

property "home" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)